godot/globals/player_status/player_status.gd

```diff
@@ -0,0 +1,14 @@
+class_name HealthSystem extends Node
+
+##Variable que almacena la vida del jugador
+var health: int = 100
+## Variable para indicar si se inicializo la vida del jugador
+var health_start: bool = false
```

Comment:
Una opción para no tener una variable extra que indique si se ha inicializado `health` es iniciarla en `null`, que indica justamente eso.
```
##Variable que almacena la vida del jugador
var health: int = null
```

Es más, si no se asigna nada, el valor inicial será null:
```
##Variable que almacena la vida del jugador
var health: int
```

Y luego:
-  `if not health` será verdadero cuando `health` sea null o 0 (cero)
- `if health == null` será verdadero cuando `health` sea null

Reply:
Al probar me di cuenta de que estaba equivocado: al iniciar una variable indicando el tipo int con `var health: int` el valor inicial es cero.

Reply:
Dejo en #164 un commit que muestra cómo usar la variable global directamente, sin tener que estar copiando la variable local y la global antes y después de cada combate.